lib/clamp/command.rb

```diff
@@ -80,6 +80,14 @@ def help
       self.class.help(invocation_path)
     end
 
+    # Get parsed options as Hash
+    #
+    def options
```

Comment:
I reckon `option_values` would be a better name for this method. For instance, `--flavour` is the option, `"strawberry"` is the value it's been given.

Reply:
> I reckon `option_values` would be a better name for this method.

No: there is a Hash. For "option values" you should call `.options.values`. Or `options[:flavour] #=> "strawberry"`.